### PR TITLE
Publication output

### PR DESCRIPTION
**Pull request by abhisrkckl**

### CODE REVIEW

File: src/pint/models/binary_ell1.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def __init__(self):
             floatParameter(
                 name="EPS1",
                 units="",
-                description="First Laplace-Lagrange parameter, ECC x sin(OM) for ELL1 model",
+                description="First Laplace-Lagrange parameter",
```
Comment (paulray) [inline src/pint/models/binary_ell1.py:120]: Why remove the specificity? Isn't it helpful?

File: src/pint/models/frequency_dependent.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def __init__(self):
                 name="FD1",
                 units="second",
                 value=0.0,
-                description="Coefficient of delay as a polynomial function of log-frequency",
+                description="Coefficient of frequency-dependent delay",
```
Comment (paulray) [inline src/pint/models/frequency_dependent.py:39]: Again, why make the description *less* specific?

File: src/pint/models/noise_model.py [hunk 3]
```diff
@@ -307,9 +301,7 @@ def __init__(
                 name="ECORR",
                 units="us",
                 aliases=["TNECORR"],
-                description="An error term added that"
-                " correlated all TOAs in an"
-                " observing epoch.",
+                description="An error term that correlates all TOAs in an observing epoch.",
```
Comment (paulray) [inline src/pint/models/noise_model.py:304]: Perhaps "An error term that is correlated among all TOAs in an observing epoch"?

File: src/pint/models/pulsar_binary.py [hunk 4]
```diff
@@ -102,7 +102,9 @@ def __init__(self):
         )
         self.add_param(
             floatParameter(
-                name="A1", units=ls, description="Projected semi-major axis, a*sin(i)"
+                name="A1",
+                units=ls,
+                description="Projected semi-major axis of pulsar orbit",
```
Comment (paulray) [inline src/pint/models/pulsar_binary.py:107]: Why not leave in the specific definition?

File: src/pint/output/publish.py [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+    if not include_swx:
+        exclude_components.append(SolarWindDispersionX)
+    if not include_tzr:
+        exclude_components.append(AbsPhase)
+
+    with StringIO("w") as tex:
+        tex.write("\\documentclass{article}\n")
+        tex.write("\\begin{document}\n")
+
+        tex.write("\\begin{table}\n")
```
Comment (paulray) [inline src/pint/output/publish.py:151]: Perhaps we should be using AASTex deluxetable environment?

**Comment by paulray** [comment]
Can you add an example of the rendered output PDF, here in the PR?

File: src/pint/models/binary_ell1.py (hunk 1 above)
Comment (abhisrkckl) [reply src/pint/models/binary_ell1.py:120]: The long descriptions were causing the summary table to be too wide.

**Comment by abhisrkckl** [comment]
Here is a wideband example:
```
$ pintpublish J1614-2230_NANOGrav_12yv3.wb.gls.par J1614-2230_NANOGrav_12yv3.wb.tim --outfile J1614-2230_NANOGrav_12yv3.wb.tex 
$ pdflatex J1614-2230_NANOGrav_12yv3.wb.tex
```
[J1614-2230_NANOGrav_12yv3.wb.pdf](https://github.com/nanograv/PINT/files/12410086/J1614-2230_NANOGrav_12yv3.wb.pdf)

**Comment by abhisrkckl** [comment]
Here is a narrowband example:
```
$ pintpublish B1855+09_NANOGrav_9yv1.gls.par B1855+09_NANOGrav_9yv1.tim --outfile B1855+09_NANOGrav_9yv1.tex 
$ pdflatex B1855+09_NANOGrav_9yv1.tex
```
[B1855+09_NANOGrav_9yv1.pdf](https://github.com/nanograv/PINT/files/12410116/B1855%2B09_NANOGrav_9yv1.pdf)

**Comment by abhisrkckl** [comment]
I have implemented latex formatting for numbers and units. Now the output looks like this
image

File: src/pint/models/binary_ell1.py (hunk 1 above)
Comment (abhisrkckl) [reply src/pint/models/binary_ell1.py:120]: added the expression again.

File: src/pint/models/frequency_dependent.py (hunk 2 above)
Comment (abhisrkckl) [reply src/pint/models/frequency_dependent.py:39]: Changed this.

**Comment by paulray** [comment]
I like the LaTeX formatting!  The only change I'd make is us ApJ style of "mas yr$^{-1}$" instead of the fraction.

**Comment by abhisrkckl** [comment]
Disabled fraction.
image

File: src/pint/models/pulsar_binary.py (hunk 4 above)
Comment (abhisrkckl) [reply src/pint/models/pulsar_binary.py]: reverted.

File: src/pint/models/noise_model.py (hunk 3 above)
Comment (abhisrkckl) [reply src/pint/models/noise_model.py]: changed.

File: src/pint/output/publish.py (hunk 5 above)
Comment (abhisrkckl) [reply src/pint/output/publish.py:157]: That requires importing a third-party sty file. I think the output should work on a minimal latex installation.

**Pull request merged**